bundles/ch.elexis.core.mail.ui/src/ch/elexis/core/mail/ui/preference/TextTemplates.java

```diff
@@ -114,6 +136,83 @@ public void selectionChanged(SelectionChangedEvent event) {
 		templateComposite = new TextTemplateComposite(parentComposite, SWT.NONE);
 		templateComposite.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true, 2, 1));
 
+		Label attachmentTitle = new Label(parentComposite, SWT.NONE);
+		attachmentTitle.setText("Anhänge");
+
+		Composite composite = new Composite(parentComposite, SWT.NONE);
+		composite.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false, 2, 1));
+		composite.setLayout(new GridLayout(2, false));
+
+		tableViewer = new TableViewer(composite, SWT.BORDER | SWT.FULL_SELECTION | SWT.V_SCROLL);
+		Table table = tableViewer.getTable();
+		ColumnViewerToolTipSupport.enableFor(tableViewer);
+		table.setLinesVisible(true);
+		tableViewer.setContentProvider(ArrayContentProvider.getInstance());
+		tableViewer.setInput(list);
+		table.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true));
+		tableViewer.addDoubleClickListener(new IDoubleClickListener() {
+
+			@Override
+			public void doubleClick(DoubleClickEvent event) {
+				int index = tableViewer.getTable().getSelectionIndex();
+
+				IStructuredSelection selectedTextTemplate = templatesViewer.getStructuredSelection();
+				ITextTemplate template = (ITextTemplate) selectedTextTemplate.getFirstElement();
+
+				IBlobSecondary textTemplate = CoreModelServiceHolder.get()
+						.load(NAMED_BLOB_PREFIX + template.getId(), IBlobSecondary.class).orElse(null);
+
+				if (textTemplate != null) {
+					byte[] attachmentData = textTemplate.getContent();
+					try {
+						List<SerializableFile> fileList = SerializableFileUtil.deserializeData(attachmentData);
+
+						try {
+							Path temp = Files.createTempFile(fileList.get(index).getName(), ".pdf");
+
+							Files.write(temp, fileList.get(index).getData());
+							Program.launch(temp.toString());
+						} catch (IOException e) {
+							e.printStackTrace();
+							MessageDialog.openError(getShell(), "Error", "Das Dokument konnte nicht geladen werden");
```

Comment:
der stacktrace geht ins leere, den siehst du nur in der IDE oder der console! Log in doch! (ebenso unten).